chcemvediet/apps/inforequests/admin.py

```diff
@@ -158,6 +158,9 @@ class InforequestAdmin(admin.ModelAdmin):
     inlines = [
             BranchInline,
             ]
+    actions = [
+            u'delete_selected',
```

Comment:
@viliambalaz IMHO nie je dobre to takto miesat, ze pri niektorych modeloch pouzivame nase `delete_selected` a pri niektorych built-in `delete_selected`. Bude z toho chaos do buducnosti. Lepsie je pri vsetkych modeloch pouzivat iba nase `delete_selected`, aby sme sa pri buducich zmenach mohli spolahnut, ze vsetko funguje vsade rovnako.

Na druhej strane pri tychto modeloch su `delete_warnings` aj `delete_constraints` prazdne, takze je zbytocne definovat pre ne prazdne metody. 

Nastal teda asi cas naprv implementovat vseobecny mixin pre nasu implementaciu `delete_selected`, ktory bude v sebe defaultne obsahovat prazdne `delete_warnings` a `delete_constraints`, a ktory poriesi vsetko okolo deletovania. Potom tu iba tento mixin pouzijeme, bez nutnosti nieco defnovat.

Mixim by mal riesit vsetky veci okolo `delete_warnings` a `delete_constraints`, tak aby potom pri kazdom modeli stacilo pouzit ten mixin, overridnut `delete_warnings` a `delete_constraints` a deletovanie bude fungovat.

Logika specificka pre konkretny model, ako napriklad snooze pri `ActionAdmin`, nebude v mixine, ale ostane nadalej iba model admine. Model admin overridne prislusne metody a doplni ich.

Pls mixin sprav v separatnom PR, aplikuj ho na modely, pre ktore mame deletovanie uz implementovane. A az potom po mergnuti PR s mixinom, sa vratime sem a upravime tento PR aby pouzival mixin aj pre tieto modely.